pvsite_datamodel/read/inverters.py

```diff
@@ -0,0 +1,25 @@
+"""Read inverters functions."""
+import logging
+import uuid
+from typing import List, Optional
+
+from sqlalchemy.orm import Session
+
+from pvsite_datamodel.sqlmodels import ClientSQL, InverterSQL
+
+logger = logging.getLogger(__name__)
+
+
+def get_inverters_for_client(session: Session, client_uuid: uuid.UUID) -> List[InverterSQL]:
+    """Get the inverters that have been linked via Enode to a specific client.
+
+    :param session: database session
+    :param client_uuid: the client uuid
+    """
+    query = session.query(ClientSQL).filter(ClientSQL.client_uuid == client_uuid).join(InverterSQL)
+
+    client: Optional[ClientSQL] = query.first()
+    if client is None:
+        return []
+
+    return client.inverters
```

Comment:
This is understandable--this function is mainly a remnant of the previous schema where it was more complex to get all inverters for a client. Given that we're certainly going to acquire this data in the API, using a helper function like this could end up making the code more readable.

Though this may be removed after addressing your comments below. Will edit later.